crates/astria-sequencer/src/sequence/action.rs

```diff
@@ -128,3 +126,7 @@ impl ActionHandler for Action {
         Ok(())
     }
 }
+
+pub(crate) fn calculate_fee(data: &[u8]) -> Option<Balance> {
```

Comment:
Suggestion for tests:

1. empty data = empty fee?
2. one byte = one "balance"?
3. 10 bytes = 10 "balance"?